src/Symfony/Component/Asset/composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "symfony/asset",
+    "type": "library",
+    "description": "Symfony Asset Component",
+    "keywords": [],
+    "homepage": "http://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "http://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=5.3.3"
+    },
+    "suggest": {
+        "symfony/http-foundation": ""
+    },
+    "require-dev": {
+        "symfony/http-foundation": "~2.4"
+    },
+    "autoload": {
+        "psr-4": { "Symfony\\Component\\Asset\\": "Symfony/Component/Asset" }
```

Comment:
this should be ``"psr-4": { "Symfony\\Component\\Asset\\": "" }``, because the ``Symfony\\Component\\Asset`` namespace is at the root of the subtree package